ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html

```diff
@@ -29,7 +29,10 @@
 </div>
 <sly data-sly-test="${!isEdgeDeliveryRequest}">
 <sly data-sly-test="${wcmmode.edit}" data-sly-use.allowed="com.day.cq.wcm.foundation.AllowedComponentList"></sly>
-<form data-sly-use.grid="com.day.cq.wcm.foundation.model.responsivegrid.ResponsiveGrid"
+
+<form data-sly-use.container="com.adobe.cq.forms.core.components.models.form.FormContainer"
```

Comment:
form container is already available, why are you instantiating it twice.

Reply:
done